core/chains/evm/chain_set.go

```diff
@@ -40,29 +40,20 @@ type ChainConfigUpdater func(*types.ChainCfg) error
 //go:generate mockery --quiet --name ChainSet --output ./mocks/ --case=underscore
 type ChainSet interface {
 	services.ServiceCtx
-	Get(id *big.Int) (Chain, error)
+	chains.DBChainSet[utils.Big, *types.ChainCfg]
```

Comment:
am i following this correctly: the chain set for ethereum would include goerli, seopolia, mainnet, etc. ? it's a class that encompasses all deployed versions of a blockchain?

Reply:
In practice, yes, each ChainSet holds all the Chains for one "family", which for EVM includes all those variants.
This all becomes much simpler after https://github.com/smartcontractkit/chainlink/pull/8434

Reply:
Yes that is correct. Plus polygon, fantom, all evm chains